lib/pdf_fill/forms/va210781a.rb

```diff
@@ -266,69 +258,99 @@ def expand_incident_location(incident)
         incident_location = incident['incidentLocation']
         return if incident_location.blank?
 
-        if incident_location.length <= 90
-          s_location = incident_location.scan(/(.{1,30})(\s+|$)/)
-          split_incident_location = {
-            'firstRow' => s_location.length.positive? ? s_location[0][0] : '',
-            'secondRow' => s_location.length > 1 ? s_location[1][0] : '',
-            'thirdRow' => s_location.length > 2 ? s_location[2][0] : ''
-          }
-          incident['incidentLocation'] = split_incident_location
+        split_incident_location = {}
+        s_location = incident_location.scan(/(.{1,30})(\s+|$)/)
 
-        else
-          incident['incidentLocation'] = {}
-          incident['incidentLocation']['incidentLocationOverflow'] = PdfFill::FormValue.new('', incident_location)
+        s_location.each_with_index do |row, index|
+          split_incident_location["row#{index}"] = row[0]
         end
+
+        incident['incidentLocation'] = split_incident_location
       end
 
       def expand_incident_unit_assignment(incident)
         incident_unit_assignment = incident['unitAssigned']
         return if incident_unit_assignment.blank?
 
-        if incident_unit_assignment.length <= 90
-          s_incident_unit_assignment = incident_unit_assignment.scan(/(.{1,30})(\s+|$)/)
-          split_incident_unit_assignment = {
-            'firstRow' => s_incident_unit_assignment.length.positive? ? s_incident_unit_assignment[0][0] : '',
-            'secondRow' => s_incident_unit_assignment.length > 1 ? s_incident_unit_assignment[1][0] : '',
-            'thirdRow' => s_incident_unit_assignment.length > 2 ? s_incident_unit_assignment[2][0] : ''
-          }
-          incident['unitAssigned'] = split_incident_unit_assignment
-        else
-          incident['unitAssigned'] = {}
-          incident['unitAssigned']['unitAssignedOverflow'] = PdfFill::FormValue.new('', incident_unit_assignment)
+        split_incident_unit_assignment = {}
+        s_incident_unit_assignment = incident_unit_assignment.scan(/(.{1,30})(\s+|$)/)
+
+        s_incident_unit_assignment.each_with_index do |row, index|
+          split_incident_unit_assignment["row#{index}"] = row[0]
         end
+
+        incident['unitAssigned'] = split_incident_unit_assignment
+      end
+
+      def get_unit_date_extras(unit_assigned_dates)
```

Comment:
What does `extras` refer to in these method names?

Reply:
My understanding is that `extras` refer to possibility of multiple / array records for it. @hopelam  can add more explanation if needed.

Reply:
"extras" refers to the formatting of the data on the overflow pages.  I will rename to make them more clear.